BudgetTracker.Business/Budgeting/Budget.cs

```diff
@@ -54,6 +76,7 @@ public class Budget
         /// <summary>
         /// The parent budget of which this is a sub-budget.
         /// </summary>
+        [JsonIgnore]

```

Comment:
Why ignore this?

Reply:
To avoid recursive serialization. Honestly, the only time we should ever serialize it to JSON is when we are printing it out to debug.